index.js

```diff
@@ -58,7 +58,7 @@ module.exports = (opts, minimistOpts) => {
 
 	const showHelp = code => {
 		console.log(help);
-		process.exit(code || 0);
+		process.exit(code || 2);
```

Comment:
Then you need to change it to `process.exit(typeof code === 'number' ? code : 2);`

Otherwise if the user tries to use `0` it will go to `2`, which is wrong.